MIGRATION.md

```diff
@@ -17,6 +17,7 @@
     - [Babel mode v7 exclusively](#babel-mode-v7-exclusively)
     - [Importing plain markdown files with `transcludeMarkdown` has changed](#importing-plain-markdown-files-with-transcludemarkdown-has-changed)
     - [7.0 feature flags removed](#70-feature-flags-removed)
+    - [Story context is prepared before for supporting fine grained updates](#story-context-is-prepared-before-for-supporting-fine-grained-updates)
```

Comment:
This did not have a TOC before my vscode addon picked it up and added it.  But I think this change isn't the most general for all users, so I moved it from the top to lower on the list.  I honestly have a bit of a hard time understanding what users need to do based on the migration instructions, but maybe others do.